### PR TITLE
Add request demo buttons to use case pages

### PR DESCRIPTION
**Pull request by mnaumanali94**

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit 3a62ed98ba98a509e14c28e8b5aeb983f8e3ff31

https://deploy-preview-1602--stoplightio.netlify.app

File: netlify/landings/spectral.yaml [hunk 1]
```diff
@@ -11,27 +11,22 @@ hero:
...
       large: true
-    - color: purple
-      href: 'https://youtube.com/watch?v=IJtFI9mkZ8Y'
-      title: Watch the Video
-      icon: ['fad', 'video']
-      type: video
-      large: true
     - color: indigo
       href: 'https://meta.stoplight.io/docs/spectral/README.md'
       title: Read the Docs
```
Comment (paulatulis) [inline netlify/landings/spectral.yaml:26]: Do you want to change this to 'View Docs' to stay consistent with the change you made for the Spectral CTA?

File: netlify/pages/pricing.yaml [hunk 2]
```diff
@@ -68,7 +68,7 @@ plans:
     summary: "Scale with best-in-class support and flexible deployment for enterprises"
     button:
       color: orange
-      title: Contact Us
+      title: Book a Demo
```
Comment (paulatulis) [inline netlify/pages/pricing.yaml:71]: Does it matter that there's not a UTM for pricing page to demo page?

File: netlify/pages/pricing.yaml [hunk 3]
```diff
@@ -84,24 +84,24 @@ plans:
       - color: orange
         href: /demo
         large: true
-        title: Contact Us
+        title: Book a Demo
```
Comment (paulatulis) [inline netlify/pages/pricing.yaml:87]: Does it matter that there's not a UTM for pricing page to demo page?

File: netlify/pages/pricing.yaml [hunk 4]
```diff
@@ -171,7 +171,7 @@ docPlans:
       large: true
     - color: orange
       href: "/demo"
```
Comment (paulatulis) [inline netlify/pages/pricing.yaml:173]: Does it matter that there's not a UTM for pricing page to demo page?

File: src/components/DocPlans/index.tsx [hunk 5]
```diff
@@ -127,13 +127,25 @@ export const DocPlans: React.FunctionComponent<IDocPlans> = ({ title, plans, cat
...
+                  title="Choose Plan"
+                  large
+                  color="blue2"
+                  className="font-bold text-center ease-in active-depress"
+                />
+              </div>
+              <div className="flex justify-center pr-10">
+                <Button
+                  href={`/demo`}
+                  title="Book a Demo"
```
Comment (paulatulis) [inline src/components/DocPlans/index.tsx:142]: Does it matter that there's not a UTM for pricing page to demo page?

File: src/components/PricingPlans/index.tsx [hunk 6]
```diff
@@ -131,15 +131,15 @@ export const PricingPlans: React.FunctionComponent<IPricingPlans> = ({ color, pl
         </div>
       </Container>
 
-      <div className="flex justify-center pb-16 mx-4 sm:hidden">
+      {/*<div className="flex justify-center pb-16 mx-4 sm:hidden">
```
Comment (paulatulis) [inline src/components/PricingPlans/index.tsx:134]: I'd say if you want to remove, go ahead and delete and we can always add it back later. But up to you!

File: netlify/pages/pricing.yaml [hunk 7]
```diff
@@ -84,24 +84,24 @@ plans:
...
-      src: /images/ea.png
-    - alt: TiVo
-      src: /images/tivo.png
-    - src: /images/logo_arkea_transparent.png
-      alt: Arkea
-    - src: /images/shopgate.png
-      alt: Shopgate
-  title: >-
-    Trusted By Leading API First Companies
+# collage:
```
Comment (paulatulis) [inline netlify/pages/pricing.yaml:89]: I'd say if you want to remove, go ahead and delete and we can always add it back later. But up to you!

File: netlify/pages/pricing.yaml [hunk 8]
```diff
@@ -170,8 +154,8 @@ docPlans:
       title: Get Started
       large: true
     - color: orange
-      href: "/demo"
-      title: Contact Us
+      href: "/demo?utm_source=documentation&utm_medium=button&utm_campaign=bookdemo"
```
Comment (paulatulis) [inline netlify/pages/pricing.yaml:157]: I think source should be pricing, right?
```suggestion
      href: "/demo?utm_source=pricing&utm_medium=button&utm_campaign=bookdemo"
```

**Comment by mnaumanali94** [comment]
@paulatulis Thanks for pointing those out!

**Review by paulatulis — APPROVED** [review]

**Pull request merged**